sdv/modeler.py

```diff
@@ -355,7 +393,7 @@ def model_database(self):
                 clean_table = self.impute_table(self.tables[table])
                 self.models[table] = self.fit_model(clean_table)
 
-        except (ValueError, np.linalg.linalg.LinAlgError):
-            ValueError(MODELLING_ERROR_MESSAGE)
+        except (ValueError, np.linalg.linalg.LinAlgError) as error:
+            raise ValueError(MODELLING_ERROR_MESSAGE).with_traceback(error.__traceback__)
```

Comment:
What's the advantage of this `with_traceback` here?

Reply:
To be able to show the original traceback along our message, this way, if they want to open an issue they have also useful information beyond a polite message of `Ups! Something went wrong`.